Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/IClusterDetail.java

```diff
@@ -83,6 +83,13 @@ default String getHttpPassword() throws HDIException {
         return null;
     }
 
+    default boolean isAmbariCredentialProvided() throws IOException, HDIException, AzureCmdException {
```

Comment:
Looks not a good place for the method, since the interface is a generic, but not all the cluster has Ambari gateway.